pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+##### Description:
+
+Explain what this PR does what problem does it solve and what changes are being made to the code base.
+
+##### QA Notes (Frontend):
```

Comment:
Optional, but I would suggest to remove the `(Frontend)` text from the title because there can be a chance to have any QA notes on the backend PRs as well

Or maybe it could be named as `QA Notes (Optional)` because it can be skipped on some of the PRs?

Reply:
Changed